docs/install/kubernetes.md

```diff
@@ -3,149 +3,143 @@ title: "Kubernetes"
+* Either login or create an account on the [Ondat Portal](https://portal.ondat.io/)
+* Choose the 'Install Ondat on your cluster' or 'Add cluster' options in the UI
+* Add a Name for your cluster and where it is going to be located.  This will allow you to view the same prerequisites as are listed above
 
-    ```bash
-    export STORAGEOS_USERNAME="storageos"
-    export STORAGEOS_PASSWORD="storageos"
-    ```
+### Step 2 - Choosing the Installation Method
 
-1. Run the following  `kubectl-storageos` plugin command to install Ondat.
+You can use either the [kubectl-storageos CLI](/docs/reference/kubectl-plugin/) or [Helm 3 CLI](https://helm.sh/docs/intro/install/) to install Ondat onto your cluster.  The most common way is to use Helm due to its popularity in the Kubernetes community, but both are fully supported and described below.
 
-    ```bash
-    kubectl storageos install \
-      --include-etcd \
-      --etcd-tls-enabled \
-      --etcd-storage-class="$ETCD_STORAGECLASS" \
-      --admin-username="$STORAGEOS_USERNAME" \
-      --admin-password="$STORAGEOS_PASSWORD"
-    ```
+### Step 3a - Installing via Helm
 
-* The installation process may take a few minutes.
+The Ondat Portal UI will display the following cmd that can be used to install Ondat using Helm.
 
-#### Step 4 - Verifying Ondat Installation
+![Helm Install](/images/docs/install/HelmInstall.png)
 
-* Run the following `kubectl` commands to inspect Ondat's resources (the core components should all be in a `RUNNING` status)
+1. The first set of commands adds the Ondat Helm repository and ensures a updated local cache.
 
-    ```bash
-    kubectl get all --namespace=storageos
-    kubectl get all --namespace=storageos-etcd
-    kubectl get storageclasses | grep "storageos"
-    ```
+```bash
+helm repo add ondat https://ondat.github.io/charts && \
+helm repo update && \
+```
 
-### Option B - Using An External `etcd` Deployment
+2. The last command installs Ondat with a set of basic install parameters that are sufficient for a basic trial installation.
 
-#### Step 1 - Setup An `etcd` Cluster
+```bash
+helm install ondat ondat/ondat \
+  --namespace=storageos \
+  --create-namespace \
+  --set ondat-operator.cluster.portalManager.enabled=true \
+  --set ondat-operator.cluster.portalManager.clientId=37540b25-285c-4326-b76c-742100723ac3 \
+  --set ondat-operator.cluster.portalManager.secret=e946f84a-e6c0-4afd-9087-f9cdd6906aa5 \
+  --set ondat-operator.cluster.portalManager.apiUrl=https://portal-setup-7dy4neexbq-ew.a.run.app \
+  --set ondat-operator.cluster.portalManager.tenantId=16e51eb9-37cf-4103-9a31-9e2cdaeec373 \
+  --set etcd-cluster-operator.cluster.replicas=3 \
+  --set etcd-cluster-operator.cluster.storage=6Gi \
+  --set etcd-cluster-operator.cluster.resources.requests.cpu=100m \
+  --set etcd-cluster-operator.cluster.resources.requests.memory=300Mi
+```
```

Comment:
People very often copy&paste the cli commands without paying much attention. At the end of point 2, I would add a comment indicating that the portalManager.{clientId,secret,apiUrl,tenantId} need to be filled with their details. 
I would suggest with a warning comment. 

A different option is to put ENV vars at the beginning of the sectiong, before helm install with the details, so it is very visible that need to be filled

```
PORTAL_CLIENTID=37540b25-285c-4326-b76c-742100723ac3
PORTAL_SECRET=e946f84a-e6c0-4afd-9087-f9cdd6906aa5 \
...
helm install ondat ...
```

Reply:
The command is not meant to be copied from the documentation, it is explaining the command that the portal gives them.  To stop people copying/pasting it I will change it to a clipped screenshot and highlight that it should be copied from the portal